tools/scripts/generate-sponsors.mts

```diff
@@ -1,202 +1,108 @@
 import fetch from 'cross-fetch';
 import * as fs from 'fs';
 import * as path from 'path';
-import prettier from 'prettier';
 
 import { PACKAGES_WEBSITE } from './paths.mts';
 
-const graphqlEndpoint = 'https://api.opencollective.com/graphql/v2';
-
-const queries = {
-  account: `{
-    account(slug: "typescript-eslint") {
-      orders(status: ACTIVE, limit: 1000) {
-        totalCount
-        nodes {
-          tier {
-            slug
-          }
-          fromAccount {
-            id
-            imageUrl
-            name
-            website
-          }
-        }
-      }
-    }
-  }`,
-  collective: `{
-    collective(slug: "typescript-eslint") {
-      members(limit: 1000, role: BACKER) {
-        nodes {
-          account {
-            id
-            imageUrl
-            name
-            website
-          }
-          tier {
-            amount {
-              valueInCents
-            }
-            orders(limit: 100) {
-              nodes {
-                amount {
-                  valueInCents
-                }
-              }
-            }
-          }
-          totalDonations {
-            valueInCents
-          }
-          updatedAt
-        }
-      }
-    }
-  }`,
-};
-
-interface AccountData {
-  orders: {
-    nodes: OrderNode[];
+type MemberNodes = {
+  account: {
+    id: string;
+    imageUrl: string;
+    name: string;
+    website: string;
   };
-}
-
-interface OrderNode {
-  fromAccount: MemberAccount;
-  tier?: Tier;
-}
-
-interface Tier {
-  slug: string;
-}
-
-interface CollectiveData {
-  members: {
-    nodes: MemberNode[];
-  };
-}
-
-interface MemberNode {
-  account: MemberAccount;
-  totalDonations: {
-    valueInCents: number;
-  };
-}
-
-interface MemberAccount {
-  id: string;
-  imageUrl: string;
-  name: string;
-  website: string;
-}
+  totalDonations: { valueInCents: number };
+}[];
 
 const excludedNames = new Set([
-  'Guest', // Apparent anonymous donor equivalent without an avatar
```

Comment:
`Guest` is not a thing anymore. The anonymous donor is "Incognito" but they get removed already because they don't have a website